### PR TITLE
Feature peers

### PR DESCRIPTION
**Pull request by rds13**
This MR is useless without featureStickyTable MR.

### CODE REVIEW

File: tasks/configure-peers.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+
```
Comment (tleguern) [inline tasks/configure-peers.yml:2]: No need for the extra blank line here.

Comment (rds13) [reply tasks/configure-peers.yml]: Fixed

**Comment by tleguern** [comment]
Can you also rebase on master? The featureStickyTable MR you rely on is already merged.

**Review by ndf-dw — APPROVED** [review]
LGTM

**Review by ndf-dw — APPROVED** [review]

**Review by ndf-dw — APPROVED** [review]
LGTM

**Comment by rds13** [comment]
> Can you also rebase on master? The featureStickyTable MR you rely on is already merged.

Done.

**Pull request closed without merging**